x/tss/keeper/keeper.go

```diff
@@ -225,6 +225,59 @@ func (k Keeper) GetMaxSimultaneousSignShares(ctx sdk.Context) int64 {
 	return shares
 }
 
+// returns the signed blocks window to be considered when calculating the missed blocks percentage
+func (k Keeper) getTssSignedBlocksWindow(ctx sdk.Context) int64 {
+	var window int64
+	k.params.Get(ctx, types.KeyTssSignedBlocksWindow, &window)
+
+	return window
+}
+
+// HasMissedTooManyBlocks returns true if the given validator address missed too many blocks within
+// the block window specifiec by this module. The block window used by this function is either the
+// cosmos slashing module window or this own module's window, depending on which one is shorter.
+func (k Keeper) HasMissedTooManyBlocks(ctx sdk.Context, address sdk.ConsAddress) (bool, error) {
+	missedBlocks, ok := k.getMissedBlocksPercent(ctx, address)
+	if !ok {
+		return false, fmt.Errorf("'%s' is not a validator", address.String())
+	}
+	maxMissedPerWindow := k.GetMaxMissedBlocksPerWindow(ctx)
+
+	return missedBlocks.GT(maxMissedPerWindow), nil
```

Comment:
are these the same dimension? One seems to be percentage and one absolute blocks?

Reply:
I would say none of them are really percentages, since we do not enforce either the numerator or the denominator values to be between 0 and 100